src/components/views/messages/MFileBody.tsx

```diff
@@ -211,123 +215,132 @@ export default class MFileBody extends React.Component<IProps, IState> {
             );
         }
 
-        const showDownloadLink = this.props.tileShape || !this.props.showGenericPlaceholder;
+        if (this.props.forExport) {
+            const content = this.props.mxEvent.getContent();
+            return <span className="mx_MFileBody">
+                <a href={content.file?.url || content.url}>
+                    { placeholder }
+                </a>
+            </span>;
+        } else {
```

Comment:
given that the above `returns` we can outdent this bit to not have to indent everything